code/script.js

```diff
@@ -1,51 +1,196 @@
-setTimeout(greeting, 1000);
+  showMessage("Welcome! Let's brighten someones day with flowers &#127804; &#127802 Who would you like to send them to?", 'bot');
+};
+
+handleNameInput.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const nameInput = document.getElementById('name-input');
+  const userName = nameInput.value; 
+  userFormInputs.name = userName;
+    
+  showMessage(`I'd like to send ${userName} flowers.`, 'user');
+    setTimeout(() => showColorOptions(userName), 1000)
+  });
+
+//Question 2
+const showColorOptions = (userName) => {
+  showMessage(`That's so nice of you! Choose a color ${userName} would like.`, 'bot')
+  
+  inputWrapper.innerHTML = `
+    <button id="redBtn">Red</button>
+    <button id="yellowBtn">Yellow</button>
+    <button id="orangeBtn">Orange</button>
+  `;
+
+document.getElementById('redBtn').addEventListener('click', () => {
+    inputWrapper.innerHTML = "";
+    setTimeout(() => showTypesOptions('Red'), 500)
+  });
+
+  document.getElementById('yellowBtn').addEventListener('click', () => {
+    inputWrapper.innerHTML = "";
+    setTimeout(() => showTypesOptions('Yellow'), 500)
+  });
+
+  document.getElementById('orangeBtn').addEventListener('click', () => {
+    inputWrapper.innerHTML = "";
+    setTimeout(() => showTypesOptions('Orange'), 500)
+  });
+}; 
+
+// Question 3 
+const showTypesOptions = (flowerType) => {
+  userFormInputs.color = flowerType;
+  showMessage(`I choose ${flowerType}.`, 'user'); 
+
+    setTimeout(() => showMessage(`Great choice! What type of ${flowerType} flowers?`, 'bot'), 1000);
+  
+    setTimeout(() => {
+      if (flowerType === 'Red') {
+        inputWrapper.innerHTML = `
+        <select id="select">
+          <option value="" selected disabled> &#127801; Select a flower type: </option>
+          <option value="Rose">Rose</option>
+          <option value="Coronation">Coronation</option>
+          <option value="Poppy">Poppy</option>
+        </select>
+      `;
+      } else if (flowerType === 'Yellow') {
+        inputWrapper.innerHTML = `
+        <select id="select">
+          <option value="" selected disabled> &#127803; Select a flower type:</option>
+          <option value="Sunflower">Sunflower</option>
+          <option value="Daffodil">Daffodil</option>
+          <option value="Daisies">Daisies</option>
+        </select>
+      `;
+      } else {
+        inputWrapper.innerHTML = `
+        <select id="select">
+          <option value="" selected disabled> &#127809; Select a flower type:</option>
+          <option value="Dahlia">Dhalia</option>
+          <option value="Tulip">Tulip</option>
+          <option value="Lily">Lily</option>
+        </select>
+      `;
+      }
+      const select = document.getElementById('select');
+      select.addEventListener('change', () => {
+        inputWrapper.innerHTML = "";
+        setTimeout(() => bouquetSize(select.value), 1000);
+      });
+    }, 1500);
+  };
```

Comment:
Interesting here. We did it a bit differently. We think it is also possible to insert this variable section into the function between row 125 and 126.